test/unit/spec/common/jobManager-Spec.js

```diff
@@ -0,0 +1,415 @@
+define(['common/jobManager', 'common/jobs', 'common/props', 'common/ui', '/test/data/jobsData'], (
+    JobManager,
+    Jobs,
+    Props,
+    UI,
+    JobsData
+) => {
+    'use strict';
+
+    const JobManagerFunctions = [
+        'cancelJob',
+        'cancelJobsByStatus',
+        'retryJob',
+        'retryJobsByStatus',
+        'updateToolbarJobStatus',
+        'updateModel',
+        'viewResults',
+        'setControlPanel',
+    ];
+
+    describe('the JobManager module', () => {
+        it('Should be loaded with the right functions', () => {
+            expect(JobManager).toBeDefined();
+            expect(JobManager.make).toBeDefined();
```

Comment:
kinda redundant with the below `Function` check

Reply:
yeah -- there are a few tests that use this pattern. They should probably all be updated.